src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -106,6 +107,12 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [phoneCode, phone, extension, showExtension]);
 
+  useEffect(() => {
+    if (phoneChanged) {
+      setTouched?.(true);
+    }
+  }, [phoneChanged, setTouched]);
```

Comment:
Does this work even when `setPhoneChanged` is called with `true` a second time? Since the value won't be different, will the effect still run?

Reply:
Yep, because we only really need it to change the touched value on the initial time (that the user changes the value in the phone field). Once its touched, it stays touched until they submit the form.

Reply:
Ah right ok that works fine for this scenario, forgot that this was related to `touched`

Reply:
Do you want @leemchale-okta to merge this to his branch or create the follow up ticket and implement this for the next release?

Reply:
yeah looks like it works.  No initial field error, and no error flicker on first type. And the rest of the functionality works

Reply:
oops sorry i just merged it to my branch I missed that

Reply:
No worries, since it functions and solves the problem + the code smell, no harm in merging it. Plus it can still be reviewed in your branch if they ultimately want to remove it :)